Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -738,15 +738,26 @@ inline RefPtr<PathOperation> BuilderConverter::convertPathOperation(BuilderState
         return RayPathOperation::create(rayValue.angle()->computeDegrees(), size, rayValue.isContaining());
     }
 
-    auto referenceBox = CSSBoxType::BoxMissing;
-    RefPtr<PathOperation> operation;
-    for (auto& currentValue : downcast<CSSValueList>(value)) {
-        if (!currentValue.isValueID()) {
+    auto processSingleValue = [&](const CSSValue& singleValue) {
+        ASSERT(!is<CSSValueList>(singleValue));
+        auto referenceBox = CSSBoxType::BoxMissing;
+        RefPtr<PathOperation> operation;
+        if (!singleValue.isValueID()) {
             operation = ShapePathOperation::create(basicShapeForValue(builderState.cssToLengthConversionData(),
-                currentValue, builderState.style().effectiveZoom()));
+                singleValue, builderState.style().effectiveZoom()));
         } else
-            referenceBox = fromCSSValue<CSSBoxType>(currentValue);
-    }
+            referenceBox = fromCSSValue<CSSBoxType>(singleValue);
+        return std::make_pair(operation, referenceBox);
+    };
+
+    RefPtr<PathOperation> operation;
+    auto referenceBox = CSSBoxType::BoxMissing;
+    if (is<CSSValueList>(value)) {
+        for (auto& currentValue : downcast<CSSValueList>(value))
+            std::tie(operation, referenceBox) = processSingleValue(currentValue);
```

Comment:
This will set both operation and referenceBox for each loop iteration, potentially overwriting the value from a previous iteration.

Reply:
Yes, fixed in latest commit